content/en/logs/error_tracking/browser_and_mobile.md

```diff
@@ -44,20 +44,11 @@ If you have not setup the Datadog Browser SDK yet, follow the [in-app setup inst
    })
    ```
 
-4. You must add the following snippet near initialization to catch and submit all uncaught exceptions to Datadog:
+4. To log a caught exception yourself, you may optionally use:
 
    ```javascript
-   window.onerror = function (message, source, lineno, colno, error) {
-       datadogLogs.logger.error(error?.message || '', {
-           error: { stack: error?.stack },
-       });
-   };
-   ```
-5. To log a caught exception yourself, you may optionally use:
-
-   ```javascript
-   const e = new Error('an exception occurred');
-   datadogLogs.logger.error(e.message, {'error': {'stack': e.stack}});
+   const err = new Error('an exception occurred');
+   datadogLogs.logger.error("an error occured", {usr: {id: 123}}, err);
```

Comment:
I added a link to your new documentation in https://github.com/DataDog/documentation/pull/17416/commits/a53c9eab92597de831ad8d295c2edeedac8a85fb so that users are presented with all options. I'll make sure to merge this PR after yours.

For the screenshots, I think the blog post which is already linked does a better job at introducing the UI.

And about the workaround, as discussed, I'm instead [adding a note](https://github.com/DataDog/documentation/pull/17416/commits/ef1090e233ba59580b63176f6b05c6fa44b5ed0a) about the fact that errors have to be instances of `Error` to be considered by Error Tracking.